web/src/views/SubmissionPortal/store/index.ts

```diff
@@ -275,7 +275,9 @@ async function incrementalSaveRecord(id: string) {
   }
 
   if (hasChanged.value) {
-    await api.updateRecord(id, payload, undefined, permissions);
+    const response = await api.updateRecord(id, payload, undefined, permissions);
+    // eslint-disable-next-line consistent-return
+    return response.httpStatus;
```

Comment:
Instead of disabling linting for this line, what are your thoughts on updating the function signature to include a return type (I think it would be `Promise<number | undefined>`?

Feel free to ignore if that change would make the linter even angrier or if you disagree.

Reply:
In addition to adding this signature I had to add returns of resolved promises for all paths. 272638b26c79673dad9cbeadb16f846adbe3f9db